diff --git a/transformer_engine/common/libtransformer_engine.version b/transformer_engine/common/libtransformer_engine.version
--- a/transformer_engine/common/libtransformer_engine.version
+++ b/transformer_engine/common/libtransformer_engine.version
@@ -20,6 +20,8 @@
			*transformer_engine::CommOverlapBase*;
			*transformer_engine::CommOverlapP2PBase*;
			*transformer_engine::CommOverlapCore*;
+			*userbuffers_send*;
+			*userbuffers_recv*;
			*nvshmem_wait_on_stream*;
			*nvshmemi_init_thread*
		};
